File: gitlfs-server/src/main/java/ru/bozaro/gitlfs/server/internal/ResponseWriter.java
package ru.bozaro.gitlfs.server.internal;

import jakarta.servlet.http.HttpServletResponse;

import javax.annotation.Nonnull;
import java.io.IOException;

/**
 * HTTP response writer.
 *
 * @author Artem V. Navrotskiy
 */
public interface ResponseWriter {
  void write(@Nonnull HttpServletResponse response) throws IOException;
}

File: gitlfs-server/src/main/kotlin/ru/bozaro/gitlfs/server/internal/ResponseWriter.kt
package ru.bozaro.gitlfs.server.internal

import jakarta.servlet.http.HttpServletResponse
import java.io.IOException

/**
 * HTTP response writer.
 *
 * @author Artem V. Navrotskiy
 */
fun interface ResponseWriter {
    @Throws(IOException::class)
    fun write(response: HttpServletResponse)
}
